packages/features/bookings/lib/handleNewBooking/test/reschedule.test.ts

```diff
@@ -604,5 +613,700 @@ describe("handleNewBooking", () => {
+            method: "POST",
+            body: mockBookingData,
+          });
+
+          const createdBooking = await handleNewBooking(req);
+          expect(createdBooking.responses).toContain({
+            email: booker.email,
+            name: booker.name,
+          });
+
+          await expectBookingInDBToBeRescheduledFromTo({
+            from: {
+              uid: uidOfBookingToBeRescheduled,
+            },
+            to: {
+              description: "",
+              // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+              uid: createdBooking.uid!,
+              eventTypeId: mockBookingData.eventTypeId,
+              // Rescheduled booking sill stays in pending state
+              status: BookingStatus.PENDING,
+              location: BookingLocations.CalVideo,
+              responses: expect.objectContaining({
+                email: booker.email,
+                name: booker.name,
+              }),
+              references: [
+                {
+                  type: appStoreMetadata.dailyvideo.type,
+                  uid: "MOCK_ID",
+                  meetingId: "MOCK_ID",
+                  meetingPassword: "MOCK_PASS",
+                  meetingUrl: "http://mock-dailyvideo.example.com",
+                },
+                {
+                  type: appStoreMetadata.googlecalendar.type,
+                  uid: "MOCK_ID",
+                  meetingId: "MOCK_ID",
+                  meetingPassword: "MOCK_PASSWORD",
+                  meetingUrl: "https://UNUSED_URL",
+                  externalCalendarId: "MOCK_EXTERNAL_CALENDAR_ID",
+                },
+              ],
+            },
+          });
+
+          expectWorkflowToBeTriggered();
+
+          expectBookingRequestedEmails({
+            booker,
+            organizer,
+            emails,
+          });
+
+          expectBookingRequestedWebhookToHaveBeenFired({
+            booker,
+            organizer,
+            location: BookingLocations.CalVideo,
+            subscriberUrl,
+            eventType: scenarioData.eventTypes[0],
+          });
+
+          expectSuccessfulVideoMeetingDeletionInCalendar(videoMock, {
+            bookingRef: {
+              type: appStoreMetadata.dailyvideo.type,
+              uid: "MOCK_ID",
+              meetingId: "MOCK_ID",
+              meetingPassword: "MOCK_PASS",
+              meetingUrl: "http://mock-dailyvideo.example.com",
+            },
+          });
+
+          expectSuccessfulCalendarEventDeletionInCalendar(calendarMock, {
+            externalCalendarId: "MOCK_EXTERNAL_CALENDAR_ID",
+            calEvent: {
+              videoCallData: expect.objectContaining({
+                url: "http://mock-dailyvideo.example.com",
+              }),
+            },
+            uid: "MOCK_ID",
+          });
+        },
+        timeout
+      );
+
+      // eslint-disable-next-line playwright/no-skipped-test
+      test.skip(
```

Comment:
Wrote the following 2 tests but they were failing as there is bug in identifying properly when an organizer reschedules the booking. Fill fix it in a followup PR as that's taking time.